src/Controller/AdminController.php

```diff
@@ -39,33 +39,26 @@ public function __construct(EventDispatcherInterface $eventDispatcher)
         $this->eventDispatcher = $eventDispatcher;
     }
 
-    /**
-     * @param $object
-     * @param $key
-     * @param $fielddefinition Data
-     * @param $objectFromVersion
-     * @param int $level
-     */
-    private function getDiffDataForField($object, $key, $fielddefinition, $objectFromVersion, $level = 0)
+    private function getDiffDataForField(Concrete $object, int | string $key, Data $fielddefinition)
```

Comment:
```suggestion
    private function getDiffDataForField(Concrete $object, int|string $key, Data $fielddefinition)
```
todo: check cs-fixer